src/abstractdataframe/reshape.jl

```diff
@@ -18,11 +18,16 @@ Stacks a DataFrame; convert from a wide to long format
 
 
 ```julia
-stack(df::AbstractDataFrame, measure_vars, id_vars)
-stack(df::AbstractDataFrame, measure_vars)
-stack(df::AbstractDataFrame)
-melt(df::AbstractDataFrame, id_vars, measure_vars)
-melt(df::AbstractDataFrame, id_vars)
+stack(df::AbstractDataFrame, measure_vars, id_vars;
```

Comment:
Please write `[measure_vars], [id_vars]` and get rid of the second definition. Same for `melt`. This is getting too verbose for my taste.